consensus/polybft/stake_manager.go

```diff
@@ -109,45 +106,32 @@ func (s *stakeManager) PostBlock(req *PostBlockRequest) error {
 	s.logger.Debug("Gotten transfer (stake changed) events from logs on block",
 		"eventsNum", len(events), "block", req.FullBlock.Block.Number())
 
+	for _, e := range events {
```

Comment:
You can log this in the already existing for loop where we update the `stakeMap`.

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1461/commits/b5951c8cbcae5029aeebfae91f42a8d96762bb4d